ac/ac-quiz/src/Dashboard/CountDashboard.jsx

```diff
@@ -3,18 +3,27 @@
 
 import * as React from 'react';
 
-import {
-  CountChart,
-  type LogDBT,
-  type dashboardViewerPropsT
-} from 'frog-utils';
-
-const Viewer = (props: dashboardViewerPropsT) => {
-  const {
-    data: { answers },
-    config
-  } = props;
-  if (!config) {
+import { CountChart, type LogDBT, type ActivityDBT } from 'frog-utils';
+
+const Viewer = ({ state: { answerCounts, questions } }: any) => (
+  <React.Fragment>
+    {(questions || []).map((q, qIndex) => (
+      <CountChart
+        key={qIndex}
+        title={q.question}
+        vAxis="Possible answers"
+        hAxis="Number of answers"
+        categories={q.answers.map(x => x.choice)}
+        data={answerCounts[qIndex]}
+      />
+    ))}
+  </React.Fragment>
+);
+
+const prepareDisplay = (state: any, activity: ActivityDBT) => {
```

Comment:
I think you have got the renaming of this function only half done. For example I observe that the count dashboard for ac-quiz works in preview perfectly, but not in a real session.